pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -91,16 +92,16 @@ let
 
   self = stdenv.mkDerivation rec {
     pname = "fwupd";
-    version = "1.5.12";
+    version = "1.6.3";
 
     # libfwupd goes to lib
     # daemon, plug-ins and libfwupdplugin go to out
     # CLI programs go to out
     outputs = [ "out" "lib" "dev" "devdoc" "man" "installedTests" ];
 
     src = fetchurl {
-      url = "https://people.freedesktop.org/~hughsient/releases/fwupd-${version}.tar.xz";
-      sha256 = "sha256-BluwLlm6s/2H/USARQpAvDR0+X8WP/q0h8VvxA6Qftc=";
+      url = "https://people.freedesktop.org/~hughsient/releases/${pname}-${version}.tar.xz";
```

Comment:
We shouldn't change URLs to contain more variables then necessary. It makes it easier to compare them to upstream ones or manually visit them.